tests/conftest.py

```diff
@@ -393,14 +393,19 @@ def telemetry_writer():
     telemetry_writer = TelemetryWriter(is_periodic=False)
     telemetry_writer.enable()
 
+    # main telemetry_writer must be disabled to avoid conflicts with the test telemetry_writer
+    ddtrace.internal.telemetry.telemetry_writer.disable()
```

Comment:
Should we wrap this in a try-finally block? If the global telemetry_writer is left in disabled state it could cause side effects

Reply:
done